lib/internal/errors.js

```diff
@@ -54,7 +61,6 @@ function message(key, args) {
 // Utility function for registering the error codes. Only used here. Exported
 // *only* to allow for testing.
 function E(sym, val) {
-  assert(messages.has(sym) === false, `Error symbol: ${sym} was already used.`);
```

Comment:
I'm fine with removing it. I was just curious. The risk of reusing the same code is low if we keep them together and in alphabetical order.